src/pages/apiCatalogue/apiCatalogue.page.jsx

```diff
@@ -27,32 +31,54 @@ const ApiCataloguePage = ({ history, currentUser: user }) => {
     order: "DESC"
   });
 
-  const radioData = {
-    legend: "Filter By",
-    formName: "filterApis",
-    values: ["All APIs", "Active APIs", "Inactive APIs"]
-  }
-
   const resetState = () => {
     setError(null);
     setIsLoaded(false);
     setApis([]);
   }
 
-  const updateApiFilter = (newApiFilter) => {
-    var publishedState = ""
-    if(newApiFilter === "All APIs"){
-      publishedState = "ALL";
-    } else {
-      publishedState = `${newApiFilter === "Active APIs" ? "" : "UN"}PUBLISHED`
+  const apiParamsOptions = {
+    // From SwaggerHub Registry API Documentation
+    state: {
+      "All APIs": "ALL",
+      "Active APIs": "PUBLISHED",
+      "Inactive APIs": "UNPUBLISHED"
+    },
+    sort: {
+      "Last Modified": "UPDATED",
+      "Relevance": "BEST_MATCH",
+      "Title": "TITLE"
+    },
+    order: {
+      "Descending": "DESC",
```

Comment:
I'm not sure if using Descending and Ascending is going to be user-friendly, maybe we could do something like A-Z and Z-A?